openapi/index.yaml

```diff
@@ -328,21 +359,15 @@ definitions:
         minLength: 1
       updated_at:
         $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/master/openapi/definitions.yaml#/Timestamp"
-      family_members:
-        $ref: "#/definitions/FamilyMembers"
-      max_amount:
-        $ref: "#/definitions/MaxBonusAmount"
-      max_tax_benefit:
-        $ref: "#/definitions/MaxBonusTaxBenefit"
+      dsu:
```

Comment:
I think it must be something like "dsu_request" because inside you have an "id" field that refers to the request not to the dsu itself.